### ISSUE TITLE
[Ch 12][Notebook][Major] Loss 노트 "숫자로 감 잡기" 표 — K=5 softmax·손실 값이 계산과 다름 (4행 중 3행)

### ISSUE THREAD

**Issue by sy2399** [yoon-gu/neuqes-101#161]
## 대상

- 기준 커밋: `61a71df` (origin/master)
- 파일: `12_bert_multiclass/12_bert_multiclass.ipynb` c3 (📐 Loss 노트 "숫자로 감 잡기 (K=5, 정답=클래스 4)" 표), `pages/12-bert_multiclass.md:61-63`, `book/chapters/ch12_bert_multiclass.tex:172-174`
- 환경: 정적 검토 + 손계산 검산. **Colab 재실행 미수행.** Ch 16 검수(#40) 중 같은 표를 검산하다 발견한 **범위 밖 건** 입니다.

## 실제 결과

표의 4행 중 3행이 softmax 계산과 맞지 않습니다. 첫 행(균등)만 맞습니다.

| logits $(z_0..z_4)$ | 표의 $\hat p_4$ | 표의 손실 | **실제** $\hat p_4$ | **실제** 손실 |
|---|---|---|---|---|
| $(0,0,0,0,0)$ | 0.200 | 1.609 | 0.200 | 1.609 ✅ |
| $(0,0,0,0,2)$ | 0.541 | 0.615 | **0.649** | **0.433** |
| $(0,0,0,0,5)$ | 0.985 | 0.015 | **0.974** | **0.027** |
| $(5,0,0,0,0)$ | 0.005 | 5.310 | **0.007** | **5.027** |

$e^2/(e^2+4) = 7.389/11.389 = 0.649$, $-\ln 0.649 = 0.433$. 나머지도 같은 식입니다.

## 기대 결과

```
| $(0, 0, 0, 0, 2)$ | $0.649$ | 0.433 |
| $(0, 0, 0, 0, 5)$ | $0.974$ | 0.027 |
| $(5, 0, 0, 0, 0)$ | $0.007$ | **5.027** ← 자신 있게 틀린 케이스 |
```

## 재현 절차

```python
import math
def loss(z, ci): s = sum(math.exp(v) for v in z); p = math.exp(z[ci]) / s; return round(p, 3), round(-math.log(p), 3)
loss([0,0,0,0,2], 4), loss([0,0,0,0,5], 4), loss([5,0,0,0,0], 4)
# ((0.649, 0.433), (0.974, 0.027), (0.007, 5.027))
```

## 추가 확인

Ch 16 의 K=7 표(#40 의 결함 이슈 참조)가 이 표를 옮겨 오며 K 만 바꾼 형태라 같이 틀려 있습니다. 두 챕터를 함께 고쳐야 "Ch 12 와 동일" 이라는 Ch 16 의 참조가 성립합니다.

## 전파 범위

```bash
grep -n '0\.541\|5\.310' 12_bert_multiclass/12_bert_multiclass.ipynb pages/12-*.md book/chapters/ch12_bert_multiclass.tex
```

| 축 | 상태 |
|---|---|
| 📓 Notebook | c3 **3행** |
| 📄 WikiDocs | `pages/12-bert_multiclass.md:61-63` **3행** |
| 📕 EPUB | `ch12_bert_multiclass.tex:172-174` **3행** — tex 손수정 |

## 심각도

**Major** — 계획서 §9 "잘못된 수식". 학습자가 검산하는 용도의 표입니다.

## 담당

Ch 12 는 통합 PR #136(`fix/ch12-review-2026-09-merged`)이 열려 있으므로 **그 브랜치에서 고치는 것이 맞습니다.** 재실행 불필요(마크다운) · 재변환 불필요(손수정) · tex 손수정.

Related to #36
